articles/143_legacy_interface_definition.md

```diff
@@ -92,11 +98,19 @@ A dynamic array can have between `0` and `N` elements of the specified type.
 
 #### Upper boundaries
 
+<div class="alert alert-info" markdown="1">
+  This feature has been added compared to the ROS 1 format.
```

Comment:
@dirk-thomas what do you mean by `has been added compared to`? That it has been added here, not like in ROS 1?

Reply:
Yes, this is an addition in this specific compared to the ROS 1 format.

Reply:
These are all new features for ROS2, maybe it could be reworded as "New for ROS2" or "Not present in ROS1" to be more explicit?  I think it makes sense as-is, but may be difficult to parse?

Reply:
See cb4739c83b64b13f92427a2d4e78d5262d12bf5c.